.envrc

```diff
@@ -1,5 +1,8 @@
 source activate ruspy_analysis
 
+# We need access to ruspy submodule
+export PYTHONPATH=$PWD/submodules/ruspy
```

Comment:
Does this replace the installing of ruspy? Maybe you can explain that to me tomorrow!

Reply:
Will do!